### PR TITLE
BETA: Register asteriamp.is-a.dev

### PR DESCRIPTION
**Pull request by optixyt0**
Added `asteriamp.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/asteriamp.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "optixyt0",
+        "email": "kaibeeley@icloud.com"
+    },
+    "record": {
+        "A": ["185.27.134.179"]
```
Comment (CuteDog5695) [inline domains/asteriamp.json:7]: cant access site "This page isn’t working. 185.27.134.179 didn’t send any data."

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/asteriamp.json (hunk 1 above)
Comment (optixyt0) [reply domains/asteriamp.json:7]: should I just send my url?

Comment (CuteDog5695) [reply domains/asteriamp.json:7]: you can

Comment (optixyt0) [reply domains/asteriamp.json:7]: asteriamp.rf.gd

Comment (optixyt0) [reply domains/asteriamp.json:7]: @CuteDog5695

**Pull request closed without merging**